tool/conda/dist/meta.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{% set version = "2.1.0.dev" %}
```

Comment:
@dcslin  need to remove the hardcode version?

Reply:
We can remove `{% set version = "2.1.0.dev" %}`.

Assume that git latest tag is always available and reliable, then we could replace all `{{ version }}` with `{{ environ.get('GIT_DESCRIBE_TAG') }}`.

One side effect is in travis build, we need to `git fetch --unshallow` to get all the tags(including latest) from remote to build environment before build starts.

Reply:
I have modified the codes (singa/cpu, singa/gpu, singa/dist) based on @dcslin suggestion:
1. remove `{% set version = "2.1.0.dev" %}`.
2. replace `{{ version }}` with `{{ environ.get('GIT_DESCRIBE_TAG') }}`.

If this is not ok, I can reverse one commit to cancel this